### PR TITLE
Testing routes

### PR DESCRIPTION
**Pull request by shubham-jangid**
I have completed the visitors route tests

### CODE REVIEW

File: server/routes/visitorRoutes.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ const visitorRoutes = express.Router();
 visitorRoutes.get('/getnotifications', getLatestNotifications);
 
 visitorRoutes.get('/getallnotifications', getAllNotifications);
+
```
Comment (s-r-aman) [inline server/routes/visitorRoutes.js]: Please, you should see what you are committing.

File: server/utils/urls.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// ------- visitiors routes urls ---------
```
Comment (s-r-aman) [inline server/utils/urls.js]: Remove comment.

File: server/utils/urls.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  getevents,
+  getallevents,
+  getsyll,
+  gettt,
+  getteacher,
+  getstudent,
+  searchstudentbyname,
+  searchstudentbyrollno,
+  verifyemail,
+};
```
Comment (s-r-aman) [inline server/utils/urls.js]: Make all the url related constants in Caps and snake case. For example:
```js
const GET_NOTIFICATIONS = '/s/visitor/getnotifications';
module.exports = { GET_NOTIFICATION }
```

File: server/tests/routes/visitors.test.js [hunk 4]
```diff
@@ -1,35 +1,231 @@
...
+  email: 'hello@gmail.com',
+  admittedIn: 'Mon Sep 11 2016 01:42:00 GMT+0530 (India Standard Time)',
+  bio:
+    'Ducimus eius est ea. Dolorem adipisci aliquid nulla autem nihil voluptas. Quis saepe numquam possimus qui et ut laboriosam est. Iste eaque omnis velit nesciunt deserunt qui.',
+  auth: {
+    provider: 'Eaque assumenda molestiae quod aut saepe et.',
+    providerId: 'voluptas',
+  },
+};
+
```
Comment (s-r-aman) [inline server/tests/routes/visitors.test.js]: Place the data in separate in seed data file.

File: server/tests/routes/visitors.test.js [hunk 5]
```diff
@@ -39,4 +235,67 @@ describe('Visitors Routes', () => {
...
+  //       expect(res.body[0]).toBeUndefined();
+  //     })
+  //     .end((err) => {
+  //       if (err) {
+  //         return done(err);
+  //       }
+  //       return done();
+  //     });
+  // });
 });
```
Comment (s-r-aman) [inline server/tests/routes/visitors.test.js]: 1. Remove all the comments from this file.
2. Split this mammoth file into multiple pieces.

File: server/routes/visitorRoutes.js (hunk 1 above)
Comment (shubham-jangid) [reply server/routes/visitorRoutes.js]: whats wrong in this file

Comment (s-r-aman) [reply server/routes/visitorRoutes.js]: The only change this file has is adding a new line.

Comment (shubham-jangid) [reply server/routes/visitorRoutes.js]: previously  there is no line break, I just introduce one to match it with the other line of code in the same file

Comment (shubham-jangid) [reply server/routes/visitorRoutes.js]: you should also see before commenting the thing.

Comment (s-r-aman) [reply server/routes/visitorRoutes.js]: Then please add this message while commiting with thr message better formatting.

**Comment by shubham-jangid** [comment]
all changes that you told me to do are done now

File: server/tests/routes/visitors.test.js [hunk 6]
```diff
@@ -1,35 +1,147 @@
...
+const saveSyllabus = saveMinimal2(Syllabus);
+const saveTimetable = saveMinimal2(TimeTable);
+const saveStudent = saveMinimal(StudentPrimary);
 afterAll(() => {
   app.server.close();
 });
 
-describe('Visitors Routes', () => {
-  test('should be a sample test', (done) => {
+describe('-- testing the visitors routes ', () => {
```
Comment (s-r-aman) [inline server/tests/routes/visitors.test.js]: What is '-- testing the visitors route' is providing that 'Visitors routes' does not?

**Comment by s-r-aman** [comment]
@shubham-jangid Delete the branch when you are done.

**Pull request merged**